docker/README.md

```diff
@@ -82,8 +82,24 @@ To support running the services locally using docker, there are three different
   > Building the containers in production mode takes longer as all node modules need to be installed and additional build
   > steps such as compiling SASS need to be performed.
 
-In order to run the services locally, you'll need to insert the appropriate values into the environment files ending with .secrets.env
-in the [env](env) folder.
+In order to run the services locally, you'll need to to rename the env files in the in the [env](env) folder to include a leading dot and removing .example. You'll need to insert the appropriate values into the environment files ending with .secrets.env. Either rename as below or grab the filles from the the gitlab repository: fish/rod-licensing-env-vars
```

Comment:
This isn't quite correct: 'Either rename as below or grab the filles from the the gitlab repository: fish/rod-licensing-env-vars' should be something like 'Copy files as follows, then get values for secret files from gitlab repo fish/rod-licensing-env-vars.'